docs/api-markdown-documenter/index.js

```diff
@@ -13,7 +13,8 @@ const path = require("path");
 const versions = require("../data/versions.json");
 const { renderApiDocumentation } = require("./render-api-documentation");
 
-const docVersions = versions.params.previousVersions.concat(versions.params.currentVersion);
+const isLocal = process.argv[2];
```

Comment:
Let's be sure to add docs for any new parameters in the file-level comment above the imports.